libraries/botbuilder-lg/src/analyzer.ts

```diff
@@ -171,9 +181,9 @@ export class Analyzer
      * @param ctx The parse tree.
      * @returns The [AnalyzerResult](xref:botbuilder-lg.AnalyzerResult) instance.
      */
-    public visitSwitchCaseBody(ctx: lp.SwitchCaseBodyContext): AnalyzerResult {
+    public visitSwitchCaseBody(ctx: SwitchCaseBodyContext): AnalyzerResult {
         const result: AnalyzerResult = new AnalyzerResult();
-        const switchCaseNodes: lp.SwitchCaseRuleContext[] = ctx.switchCaseTemplateBody().switchCaseRule();
+        const switchCaseNodes: SwitchCaseRuleContext[] = ctx.switchCaseTemplateBody().switchCaseRule();
```

Comment:
This typing doesn't look to be necessary, but with the other work going on I'm not blocking on this